docs/guide/installation.md

```diff
@@ -0,0 +1,185 @@
+# Installation
+
+-   [You Tube Tutorials](#youtube-tutorials)
+-   [Download a compiled binary](#download-a-compiled-binary)
+-   [Building from source](#building-from-the-source-code)
+-   [Run using Docker](#using-docker)
+
+## You Tube Tutorials
+
+If you're not the kind of person that likes textual tutorials, our community has made video tutorials available on You Tube.
+
+-   [Installation on Windows by Daenges](https://www.youtube.com/watch?v=BkCqOCa2uQw)
+-   [Installation on Windows + Auto AFK and More by Dexter113](https://www.youtube.com/watch?v=FxJ0KFIHDrY)
+
+## Download a compiled binary
+
+#### Windows
+
+In order to run **Minecraft Console Client** on Windows you need the following software:
+
+-   [.NET Framework 4.8 or new-er](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
```

Comment:
Either 4.0 or greater (original version)
Or 6.0 or greater (new version https://github.com/MCCTeam/Minecraft-Console-Client/pull/1674)
(Seems like documentation below states 6.0 so you can just go for 6.0 as the original version will be discontinued)

Reply:
Implemented the requested changes.